05_iigo/index.tex

```diff
@@ -948,7 +948,64 @@ \subsection{Rule Representation}
+    \item Rules-in-Use
+    
+    In IIGO we have built a system for storing and checking rules (see Section~\ref*{sec:rule_representation}). There are operational choice rules (i.e., those introduced by systems of taxation, allocation and sanctions), collective choice rules (e.g., those stating the obligations and permissions of each role as well as the islands of the archipelago). The collective choice rules available to the agents can be seen listed after powers throughout the chapter. IIGO does not employ any constitutional choice rules, that are not "hard-coded" into the system. For example, all islands always vote in elections; in deciding an allocation, the President can be the only island involved in the final decision-making process. This, of course, was a limitation of the available scope of the project.
+    
+    \item Action Arenas
+    \begin{enumerate}
+        \item The first decision arena of IIGO is that of rules surrounding appropriation and provision. The decision group consists solely of the President, however, all islands can provide information to the President by reporting the number of resources in their common pool as well as requesting a desired allocation. Here the island resource reports and allocation requests are information that the president can use to make a decision and set out the rules that we call tax and allocation. 
+        
+        \item The second decision arena of IIGO is that surrounding collective choice rules. This is the process through which islands can change the rules in play and is the main concern of the legislative branch. All islands are involved in this process by proposing rules and voting but the Speaker and the President hold special decision-making powers (Section~\ref*{sec:legislative} and Section~\ref*{sec:executive}), which are expressed through the powers they hold.
+        
+        \item The third decision arena of IIGO is that surrounding sanctions. Here the Judge is the only actor involved in the decision making of whether to and how harshly to sanction islands, as well as whether to pardon islands.
+        
+        \item The fourth decision arena of IIGO is that surrounding monitoring of role powers and elections. The two are interlinked whereby the elected actors have to participate in not only the decision making surrounding elections but also the decision making in monitoring, which in turn can alter the rules in play to oblige other actors to hold elections. Moreover, elections involve the involvement of all islands voting, thus providing information to the island responsible for the appointment of a role. It is important to note is the repetitive nature of decisions roles have to make regarding elections, followed by decisions regarding monitoring, followed by elections, e.t.c. 
```

Comment:
minor but etcetera and is abbreviated by etc. - not sure what e.t.c stands for - quick fix

Reply:
I wrote that late at night and kept staring at it thinking it doesn't look right but I just wrote off as 1AM madness haha